### PR TITLE
Transporter/reader suggestions

### PR DESCRIPTION
**Pull request by gvasquezvargas**
## What Changed?

Some suggestions to use the reader. 
In a different PR, I saw the reader was being called "EDB Transporter Writer". If that is the case, we'll need to update here as well.

### CODE REVIEW

File: product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx [hunk 1]
```diff
@@ -72,37 +72,41 @@ Set the following environment variables in `/opt/cdcreader/run-cdcreader.sh` wit
 #export QUARKUS_LOG_CATEGORY__COM_ENTERPRISEDB__LEVEL=DEBUG
 ```
 
-## Running the reader
```
Comment (gvasquezvargas) [inline product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx:75]: Moved this to the bottom of the page, as this is the command the customer will have to run _after_ configuring all values.

File: product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx [hunk 2]
```diff
@@ -72,37 +72,41 @@ Set the following environment variables in `/opt/cdcreader/run-cdcreader.sh` wit
...
-
 ### Parameters explanation
 
 #### DBZ_ID
-ID to identify the reader. Users can define the ID they think will be useful with below constraints:
 
-1. ID length should lower or equal to 255
-2. ID can only contain lowercase characters, uppercase characters, numbers, underscore(_) and hyphen(-)
-3. ID should be unique, two reader instances should not use the same ID
+This is the name you assign to a migration to identify it.
```
Comment (yidian1997) [inline product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx:79]: This DBZ_ID is assigned to a reader not a migration

File: product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx [hunk 3]
```diff
@@ -72,37 +72,41 @@ Set the following environment variables in `/opt/cdcreader/run-cdcreader.sh` wit
...
+Take into account the following limitations: 
+
+- The ID cannot be longer than 255 characters.
+- The ID can contain lowercase characters, uppercase characters, numbers, underscores(_) and hyphens(-)
+- The ID must be unique; two reader instances should not have the same ID.
 
 #### RW_SERVICE_HOST
-URL of the transporter-rw-service is always https://transporter-rw-service.biganimal.com
+
+Uncomment the line. The URL of the `transporter-rw-service` is always https://transporter-rw-service.biganimal.com.
```
Comment (yidian1997) [inline product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx:88]: Why do we need to add "Uncomment the line." here

File: product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx (hunk 2 above)
Comment (gvasquezvargas) [reply product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx:79]: Is this also the name that will later appear as the source name in the EDB Cloud Service (BigAnimal) UI? If yes, we could say: 
```suggestion
This is the name you assign to identify a reader. This name will later appear as a _source_ in the EDB Cloud Service UI. 
```
If not, we can simply say: 
```suggestion
This is the name you assign to a reader.
```

File: product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx (hunk 3 above)
Comment (gvasquezvargas) [reply product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx:88]: You are right, we are not suggesting to uncomment all other lines of the reader when configuring them, so we should not explicitly call it out here. What about this? 

```suggestion
Specifies the URL of the service that will host the migration. `transporter-rw-service` is always https://transporter-rw-service.biganimal.com.
```

File: product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx
Comment (Xander-run) [inline product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx:1]: How about changing the title to "Configuring & starting the source reader" or something similar to indicate that the startup script is included?

Comment (gvasquezvargas) [reply product_docs/docs/transporter/ba_preview/getting_started/config_reader.mdx:1]: Good point, I altered the title as well ✅

**Review by Xander-run — APPROVED** [review]
LGTM!

**Pull request merged**